src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -162,244 +231,245 @@ protected static ConfigDef baseConfigDef() {
-    ).define(
-        CONNECTION_USERNAME_CONFIG,
-        Type.STRING,
-        null,
-        Importance.MEDIUM,
-        CONNECTION_USERNAME_DOC,
-        group,
-        ++order,
-        Width.SHORT,
-        "Connection Username"
-    ).define(
-        CONNECTION_PASSWORD_CONFIG,
-        Type.PASSWORD,
-        null,
-        Importance.MEDIUM,
-        CONNECTION_PASSWORD_DOC,
-        group,
-        ++order,
-        Width.SHORT,
-        "Connection Password"
-    ).define(
-        BATCH_SIZE_CONFIG,
-        Type.INT,
-        2000,
-        Importance.MEDIUM,
-        BATCH_SIZE_DOC,
-        group,
-        ++order,
-        Width.SHORT,
-        "Batch Size"
-    ).define(
-        MAX_IN_FLIGHT_REQUESTS_CONFIG,
-        Type.INT,
-        5,
-        Importance.MEDIUM,
-        MAX_IN_FLIGHT_REQUESTS_DOC,
-        group,
-        5,
-        Width.SHORT,
-        "Max In-flight Requests"
-    ).define(
-        MAX_BUFFERED_RECORDS_CONFIG,
-        Type.INT,
-        20000,
-        Importance.LOW,
-        MAX_BUFFERED_RECORDS_DOC,
-        group,
-        ++order,
-        Width.SHORT,
-        "Max Buffered Records"
-    ).define(
-        LINGER_MS_CONFIG,
-        Type.LONG,
-        1L,
-        Importance.LOW,
-        LINGER_MS_DOC,
-        group,
-        ++order,
-        Width.SHORT,
-        "Linger (ms)"
-    ).define(
-        FLUSH_TIMEOUT_MS_CONFIG,
-        Type.LONG,
-        10000L,
-        Importance.LOW,
-        FLUSH_TIMEOUT_MS_DOC,
-        group,
-        ++order,
-        Width.SHORT,
-        "Flush Timeout (ms)"
-    ).define(
-        MAX_RETRIES_CONFIG,
-        Type.INT,
-        5,
-        Importance.LOW,
-        MAX_RETRIES_DOC,
-        group,
-        ++order,
-        Width.SHORT,
-        "Max Retries"
-    ).define(
-        RETRY_BACKOFF_MS_CONFIG,
-        Type.LONG,
-        100L,
-        Importance.LOW,
-        RETRY_BACKOFF_MS_DOC,
-        group,
-        ++order,
-        Width.SHORT,
-        "Retry Backoff (ms)"
-      ).define(
-        CONNECTION_TIMEOUT_MS_CONFIG, 
-        Type.INT, 
-        1000, 
-        Importance.LOW, 
-        CONNECTION_TIMEOUT_MS_CONFIG_DOC,
-        group, 
-        ++order, 
-        Width.SHORT, 
-        "Connection Timeout"
+    configDef
+        .define(
+            CONNECTION_URL_CONFIG,
+            Type.LIST,
+            Importance.HIGH,
+            CONNECTION_URL_DOC,
+            CONNECTOR_GROUP,
+            ++order,
+            Width.LONG,
+            CONNECTION_URL_DISPLAY
+        ).define(
+            CONNECTION_USERNAME_CONFIG,
+            Type.STRING,
+            CONNECTION_USERNAME_DEFAULT,
+            Importance.MEDIUM,
+            CONNECTION_USERNAME_DOC,
+            CONNECTOR_GROUP,
+            ++order,
+            Width.SHORT,
+            CONNECTION_USERNAME_DISPLAY
+        ).define(
+            CONNECTION_PASSWORD_CONFIG,
+            Type.PASSWORD,
+            CONNECTION_PASSWORD_DEFAULT,
```

Comment:
Should this have a default? or should it be defined with `Importance.HIGH` and a required config?

Reply:
this is not a required config